go/private/actions/stdlib.bzl

```diff
@@ -120,10 +121,13 @@ def _sdk_stdlib(go):
         root_file = go.sdk.root_file,
     )
 
+def _dirname(file):
+    return file.dirname
+
 def _build_stdlib(go):
     pkg = go.declare_directory(go, path = "pkg")
-    args = go.builder_args(go, "stdlib")
-    args.add("-out", pkg.dirname)
+    args = go.builder_args(go, "stdlib", use_path_mapping = True)
+    args.add_all("-out", [pkg], map_each = _dirname, expand_directories = False)
```

Comment:
What's the difference between this line and and `args.add("-out", pkg.dirname)`?